pkg/apis/pipeline/v1beta1/resultref_test.go

```diff
@@ -41,6 +41,17 @@ func TestNewResultReference(t *testing.T) {
 			PipelineTask: "sumTask",
 			Result:       "sumResult",
```

Comment:
could you add a test case for an object result with multiple properties? e.g. `(tasks.sumTask3.results.imageresult.digest)` and `(tasks.sumTask3.results.imageresult.tag)`

Reply:
> could you add a test case for an object result with multiple properties? e.g. `(tasks.sumTask3.results.imageresult.digest)` and `(tasks.sumTask3.results.imageresult.tag)`

Added. Thanks @lbernick !